app/src/pages/DataCatalogPage.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import DatasetExplorer from "../features/dataset_explorer/DatasetExplorer";
```

Comment:
I guess in my mind the page was called DataCatalog and within it there was the DatasetExplorer but there may also be other things (like text or images or things)

I'm kinda of inclined just to leave it as it since these layouts and components aren't really firm and we may have to totally refactor them later anyway. I'm happy to change the names if you described if that paradigm makes sense still when things are more final

Reply:
Sounds good, fine to leave as is :)